projects/make-a-vis/src/lib/mav-selection/graphic-variable-type/graphic-variable-type.component.ts

```diff
@@ -76,7 +76,7 @@ export class GraphicVariableTypeComponent implements OnInit, OnChanges {
 
   getGraphicVariableOptions() {
```

Comment:
This naming is confusing considering it doesn't actually return graphic variable options.